cli/required.go

```diff
@@ -0,0 +1,32 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func NoArgs(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return nil
+	}
+	return fmt.Errorf("unexpected argument: %s\nSee '%s --help'", args[0], cmd.CommandPath())
+}
+
+//func NoArgs() func(*cobra.Command, []string) error {
```

Comment:
Yes, but I'm not done with this file yet. It's the last remaining wip. Among other things, I need to add more argument validators. I'll clean this up as part of my linting and squashing.